README.md

```diff
@@ -370,6 +370,53 @@ $updated_contact = ChartMogul\Contact::update([
 $merged_contact = ChartMogul\Contact::merge($into_contact_uuid, $from_contact_uuid);
 ```
 
+### Customer Notes and Call Logs
+
+**List Customer Notes**
+
+```php
+$contacts = ChartMogul\CustomerNote::all([
+  'cursor' => 'aabbccdd...'
+]);
```

Comment:
Looks like a copy-paste mistake

```suggestion
$notes = ChartMogul\CustomerNote::all([
  'cursor' => 'aabbccdd...'
]);
```
Also, does this mean that this API will get all notes regardless of the customer?

Furthermore, the interface is inconsistent with the rest of the API. This is how to list all contacts in a customer:

```php
$customer = ChartMogul\Customer::retrieve("cus_52eb54c2-dea0-11ed-ac96-ef735d89fca0");
$customer->contacts(["cursor" => "aabbcc...", "per_page" => 50]);
```

Since a customer note always requires a customer, I think this kind of interface should benefit from it. The customer will always be a required prerequisite whereas with the current interface, `customer_uuid` is an additional param